site/markdownlint.json

```diff
@@ -1,10 +1,12 @@
 {
-  "default": true,
+  "default": false,
```

Comment:
what does default means in this case ?

Reply:
It toggles the enforcement of a set of rules according to https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
We're going to be piecewise enabling rules as they fit our style guide, so we're setting it to false here.